pom.xml

```diff
@@ -515,8 +515,8 @@
                 <configuration>
                     <skipCheckoutIfExists>false</skipCheckoutIfExists>
                     <connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
-                    <scmVersionType>tag</scmVersionType>
-                    <scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+                    <scmVersionType>branch</scmVersionType>
+                    <scmVersion>CHK-907-commons-refactoring</scmVersion>
```

Comment:
Before merge it, please update with new ecommerce-commons version

Reply:
Added `do not merge` label to reflect this